src/main/java/org/cactoos/list/ListIteratorNoNulls.java

```diff
@@ -97,11 +95,21 @@ public void remove() {
 
     @Override
     public void set(final T item) {
+        if (item == null) {
```

Comment:
@fabriciofx Using a static check method is not an option (i.e `Objects.requireNonNull`) here?
This check for `NULL` seems to be copy-pasted in many places.

Reply:
@andreoss since this is the main responsibility of the class to check for null, I don't think it's really a problem. Also we tend to avoid using static methods such as `Objects.requireNonNull` when possible (I agree there are case it is just better to use the Java lib of course) and here it seems it is possible. In particular, it allows us to decide what is the error message here :)